packages/split-layout/theme/lumo/vaadin-split-layout-styles.js

```diff
@@ -20,12 +20,15 @@ registerStyles(
       justify-content: center;
       width: var(--lumo-size-m);
       height: var(--lumo-size-m);
+      left: calc(50% - 0.5px);
+      top: calc(50% - 0.5px);
```

Comment:
It looks like this change here causes the handle to become misaligned when using an increased zoom level (300% used in the screenshots):

![Bildschirmfoto 2023-08-08 um 12 19 04](https://github.com/vaadin/web-components/assets/357820/8d315ef7-d33b-4f31-8faa-90ab51190975)
![Bildschirmfoto 2023-08-08 um 12 18 51](https://github.com/vaadin/web-components/assets/357820/bf70cbcb-9d9d-496c-a067-a987ffe67a74)

Reply:
It's also misaligned on a high DPI display (Macbook) without changing the zoom level.

Reply:
I suppose the 0.5px adjustment should only be done when the handle size is uneven.